packages/documentation/src/stories/components/accordion/accordion.stories.ts

```diff
@@ -99,39 +100,38 @@ export const DefaultCollapsedPanels: Story = {
   `,
 };
 
-// Used only in testing for now.
 export const Nested: Story = {
   render: () => html`
-    <post-accordion>
+    <post-accordion heading-level="4">
```

Comment:
Why did you not use the storybook logic to override args on the story level here?